### ISSUE TITLE
support range syntax from Media Queries Level 4

### ISSUE THREAD

**Issue by alexander-akait** [csstree/csstree#143]
Example:

```CSS
@media (height > 600px) {
    body { line-height: 1.4; }
}

@media (400px <= width <= 700px) {
    body { line-height: 1.4; }
}
```

**Comment by romainmenke** [comment]
This shipped in Chrome 104 recently

**Comment by lahmatiy** [comment]
Fixed in [v3.0.0](https://github.com/csstree/csstree/releases/tag/v3.0.0)
